entrypoints/puppet-lint

```diff
@@ -35,6 +35,9 @@ fi
 
 eval $(pdk env)
 
+# Remove any custom config if present
+rm -f -- .puppet-lint.rc
```

Comment:
It seems like allowing users to provide some level of custom config could be desirable in the future, but I think that would be a much larger feature. As far as I can tell, the existing Anubis validation (v1) doesn't take into account any custom config, so this would simply ensure Anubis v2 behaves the same.

Reply:
:+1: this makes sense to me!